Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/commit.py

```diff
@@ -98,6 +103,11 @@ def main(cls, args, repository, command=None, representation=None, **kwargs):
         env = os.environ
         env['COMMIT_MESSAGE_TITLE'] = issue.title if issue else ''
         env['COMMIT_MESSAGE_BUG'] = '\n'.join(cls.bug_urls(issue))
+        if args.update:
+            env['WKSCMPY_UPDATE_CHANGELOG'] = 'True'
+        elif args.update is False:
+            env['WKSCMPY_UPDATE_CHANGELOG'] = 'False'
```

Comment:
Instead of the `is False`, I think I'd go with `if args.update is not None: env['WKSCMPY_UPDATE_CHANGELOG'] = str(bool(args.update))`.